builtin/fsmonitor--daemon.c

```diff
@@ -963,6 +963,10 @@ static int do_handle_client(struct fsmonitor_daemon_state *state,
 	fsmonitor_format_response_token(&response_token,
 					&state->current_token_data->token_id,
 					state->current_token_data->batch_head);
+
+	trace2_data_string("fsmonitor", the_repository, "response/token",
+			   response_token.buf);
+	trace_printf_key(&trace_fsmonitor, "response token: %s", response_token.buf);
```

Comment:
Let's put these 2 lines after the unlock?

Reply:
Wait, I have the Trace2 message at line 973 after the reply().
I think we just need to add the `trace_printf_key()` after it and before the second reply().

Reply:
and also add the `trace_printf_key()` after 991

Reply:
See https://github.com/gitgitgadget/git/pull/923/commits/a442e287531ee1ae09b7e77eab93610d979b3069